net.sourceforge.osmo.yml

```diff
@@ -0,0 +1,124 @@
+app-id: net.sourceforge.osmo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: osmo.desktop
+rename-icon-file: osmo
+command: osmo
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /lib/pkgconfig
+  - /include
+  - /share/man
+
+modules:
+  - name: gspell
+    modules:
+      - name: enchant
+        cleanup:
+          - /bin
+        sources:
+          - type: archive
+            url: https://github.com/AbiWord/enchant/releases/download/v2.3.3/enchant-2.3.3.tar.gz
+            sha256: 3da12103f11cf49c3cf2fd2ce3017575c5321a489e5b9bfa81dd91ec413f3891
+            x-checker-data:
+              type: anitya
+              project-id: 6601
+              url-template: https://github.com/AbiWord/enchant/releases/download/v$version/enchant-$version.tar.gz
+    cleanup:
+      - /bin
+    config-opts:
+      - --disable-vala
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gspell/1.10/gspell-1.10.0.tar.xz
+        sha256: 803bb884c0215d3fd22a85d7f30423aff88d9792f05a5199d8a489a2ffaae1da
+        x-checker-data:
+          type: gnome
+          name: gspell
+
+  - name: libical
+    cleanup:
+      - /lib/cmake
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_LIBDIR=lib
+      - -DBUILD_SHARED_LIBS:BOOL=ON
+      - -DICAL_GLIB=true
+      - -DGOBJECT_INTROSPECTION=true
+      - -DICAL_GLIB_VAPI=false
+      - -DICAL_BUILD_DOCS=false
+    sources:
+      - type: archive
+        url: https://github.com/libical/libical/releases/download/v3.0.14/libical-3.0.14.tar.gz
+        sha256: 4284b780356f1dc6a01f16083e7b836e63d3815e27ed0eaaad684712357ccc8f
+        x-checker-data:
+          type: anitya
+          project-id: 1637
+          url-template: https://github.com/libical/libical/releases/download/v$version/libical-$version.tar.gz
+
+  - name: libnotify
+    buildsystem: meson
+    config-opts:
+      - -Dman=false
+      - -Dgtk_doc=false
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libnotify/-/archive/0.7.9/libnotify-0.7.9.tar.gz
+        sha256: 9bd4f5fa911d27567e7cc2d2d09d69356c16703c4e8d22c0b49a5c45651f3af0
+
+  - name: libmcrypt
+    buildsystem: autotools
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/mcrypt/files/Libmcrypt/2.5.8/libmcrypt-2.5.8.tar.gz
+        sha256: e4eb6c074bbab168ac47b947c195ff8cef9d51a211cdd18ca9c9ef34d27a373e
+      - type: shell
+        commands: autoconf -f -i
```

Comment:
```suggestion
        commands: 
          - cp -p /usr/share/automake-*/config.{sub,guess} .
          - autoconf -f -i
```

Reply:
that's to fix the build failure on aarch64

Reply:
Thanks